### ISSUE TITLE
GeneMark-ES is not found

### ISSUE THREAD

**Issue by KatharinaHoff** [gatech-genemark/ProtHint#1]
Hi Tomas!

I called ProtHint like this:

`prothint.py --threads=10 /home/katharina/git/BRAKER/example/genome.fa proteins.fasta`

gmes_petap.pl is in my path:

```
which gmes_petap.pl
/home/katharina/gm_et_linux_64/gmes_petap.pl
```

ProtHint only checks in the current directory. Maybe you could change this similar to the following if it cannot be found in current directory:

```
if shutil.which("gmes_petap.pl") is not None:
        gmes_petap = shutil.which("gmes_petap.pl")
else:
        frameinfo = getframeinfo(currentframe())
        print('Error in file ' + frameinfo.filename + ' at line ' + str(frameinfo.lineno) + ': ' + "Unable to locate gmes_petap.pl")
```
Katharina

**Comment by tomasbruna** [comment]
Hi Katharina,

I see -- this problem will apply to all other dependencies as well. It makes sense to check the path if dependencies cannot be found in the expected folder, I will add this functionality.

Tomas

**Comment by tomasbruna** [comment]
Implemented in 82ca30199e3e641244ce752c6e156dacae0bdaec and 2d4b8d47ef7f362d9a50db5b396168e0d38fef95
